scripts/common.sh

```diff
@@ -39,3 +39,11 @@ else
   SED=$(resolveCommand sed)
 fi
 export SED
+
+#
+# Parses a registry server from registry destination such as quay.io/podman/stable
+# Works with remote address forms described by https://docs.podman.io/en/stable/markdown/podman-push.1.html
+#
+parseRegistryServer () {
```

Comment:
I get that it is parsing the registry server from the destination but I think a more helpful name would be `extractRegistiryServer` as it tells readers why we are parsing and that it returns it rather than say validating it.

Reply:
Done.